tests/spread/integration/gearman-job-server/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: Integration tests for gearman-job-server
+
+execute: |
+  # Chisel a minimum number of slices to give us a runnable system that we can
+  # test in.
+  rootfs="$(install-slices gearman-job-server_bins)"
+
+  chroot "${rootfs}/" gearmand -h
```

Comment:
I wonder if there's a practical usage we could test here instead of help. If so, and if it's easy and convenient, let's please test that.

Reply:
Ah okay, this is fine if that's not simple enough.